pkg/builder/builder.go

```diff
@@ -702,6 +705,12 @@ func (b *Builder) buildRayClusterSpec(spec *rayv1.RayClusterSpec) {
 	}
 }
 
+func (b *Builder) buildJobSetSpec(jobSetSpec *jobsetapi.JobSetSpec) {
+	for _, val := range jobSetSpec.ReplicatedJobs {
+		b.buildPodSpecVolumesAndEnv(&val.Template.Spec.Template.Spec)
```

Comment:
I think you need to use `b.buildPodTemplateSpec()` to implement `requests` and `command` flags.